roles/openshift_master/tasks/main.yml

```diff
@@ -131,6 +131,23 @@
 - name: Install the systemd units
   include: systemd_units.yml
 
+- name: Checking for journald.conf
+  stat: path=/etc/systemd/journald.conf
+  register: journald_conf_file
+
+- name: Increase rate limits in journald
+  lineinfile:
+    dest: /etc/systemd/journald.conf
+    regexp: "{{ item.regexp }}"
+    line: "{{ item.line }}"
+  with_items:
+    - { regexp: '^#?RateLimitInterval=.*$', line: 'RateLimitInterval=1s' }
+    - { regexp: '^#?RateLimitBurst=.*$', line: 'RateLimitBurst=10000' }
```

Comment:
I don't see any problems making it more configurable, I've opened this PR to have a discussion happening, since I've stumbled on this problem.